src/locales/en.json

```diff
@@ -204,18 +204,18 @@
   },
   "about": {
     "intro_header": "What is the microCOVID project?",
-    "basic_description": "This is a project to quantitatively estimate the COVID risk to you from your ordinary daily activities. We trawled the scientific literature for data about the likelihood of getting COVID from different situations, and combined the data into a model that people can use. We estimate COVID risk in units of microCOVIDs, where 1 microCOVID = a one-in-a-million chance of getting COVID.",
+    "basic_description": "Calculate your COVID-19 risk from daily activities, estimated in microCOVID units (1 mC being a one-in-a-million chance of contracting COVID-19, based on the scientific literature on it).",
     "where_to_start_header": "Where do I start?",
     "data_inputs_lead": "This website contains three outputs of our investigation",
-    "data_inputs_calculator": "a <1>calculator</1> that you can use to calculate your COVID risk",
-    "data_inputs_whitepaper": "a <1>white paper</1> that explains our estimation method",
-    "data_inputs_spreadsheet": "and a <1>spreadsheet</1> to compute your COVID risk in more detail and to track your risk over time.",
-    "data_inputs_recommendation": "You should start with either the <2>calculator</2> or the <5>white paper</5>.",
+    "data_inputs_calculator": "the <1>calculator</1> used to calculate your COVID-19 risk",
+    "data_inputs_whitepaper": "the <1>white paper</1> explaining our estimation method",
+    "data_inputs_spreadsheet": "the <1>spreadsheet</1> to compute your COVID-19 risk in finer detail, and track your risk over time.",
+    "data_inputs_recommendation": "Start with either the <2>calculator</2> or the <5>white paper</5>.",
     "disclaimer": "<0>Disclaimer</0>In this tool we state our best estimate based on available evidence, even when that evidence is not conclusive. We have read a lot of experts' research, but we are not ourselves experts in this topic. This work has not been scientifically peer-reviewed. There is still a lot of uncertainty about COVID. Do not rely on this tool for medical advice. Please continue to follow government guidance.",
     "contact_us_header": "Contact us",
-    "feedback": "If you have any feedback, questions, or would like to help out, please contact us <1>here</1>.",
+    "feedback": "Provide feedback, ask questions, or help out by <1>contacting us</1>.",
     "donations_header": "Donations",
-    "donations_details": "We also welcome donations and tips, if you feel so moved. No pressure, truly. We're all volunteers and we don't currently have any funding to work on this project. Thank you in advance!"
+    "donations_details": "Donations and tips welcomed, should you feel so inclined. No pressure, truly. We're all volunteers and don't currently have any funding to work on this project. Thank you in advance!"
```

Comment:
Odd that you stripped one "we" but not the other.

Reply:
One was redundant, and this is meant to sound nice(?), rather than the disclaimer above.